sections/alert_production_pipelines.tex

```diff
@@ -219,7 +219,7 @@ \subsubsection{Output Data}
 
 \paragraph*{DiffExp Images:} Image differences derived by subtracting a TemplateCoadd from a CalExp image.
 
-\paragraph*{DIASource Databases:} \DIASources detected and measured from the DiffExps using the set of parameters described in Table~\ref{table:ap_features} will be persisted.
+\paragraph*{DIASource Databases:} \DIASources detected and measured from the DiffExps using the set of parameters described in DPDD (LSE-163) will be persisted.
```

Comment:
`\DPDD` will put in a (blue) link directly to DPDD (And add entry to references). `\citedsp[LSE-163]` will add a link to the references and insert the text `[LSE-163]`. Either way you get a link to DPDD, it just depends on the indirection you want and what you want the text to say. Other places have just used `\DPDD`.